src/main.rs

```diff
@@ -102,11 +102,13 @@ async fn poise(#[shuttle_secrets::Secrets] secret_store: SecretStore) -> Shuttle
 						let response = if error.is::<poise::CodeBlockError>() {
 							"\
 Missing code block. Please use the following markdown:
-\\`code here\\`
+`` `code here` ``
 or
-\\`\\`\\`rust
+```ansi
+`\x1b[0m`\x1b[0m`rust
 code here
-\\`\\`\\`"
+`\x1b[0m`\x1b[0m
```

Comment:
This is not copy pastable anymore now, I think?

Reply:
ANSI escape codes aren't included when you select the code block and copy it.

Reply:
Also, I just noticed that I forgot the third backtick, so I added it.

Reply:
Ohh, the code block type is ```ansi! That's very smart, it makes total sense that the escape codes aren't copied with the text in Discord, leveraging that behavior is elegant